libs/features/creature/src/creature-template/creature-template.component.html

```diff
@@ -537,66 +537,17 @@
         </div>
       </div> -->
       </form>
-      <div class="preview-container" [class.show-preview]="showItemPreview" [class.hide-preview]="!showItemPreview">
+      <div class="preview-container" [class.show-preview]="showCreaturePreview" [class.hide-preview]="!showCreaturePreview">
         <div class="transparent perfect-scrollbar">
-          <!-- TODO: fix and  enable these in the scope of https://github.com/azerothcore/Keira3/issues/3015-->
+          <!-- TODO: implement creature preview like item/quest preview -->
           <div style="color: white; margin: 10px">
             <p>Preview currently unavailable.</p>
             <p>This feature will be fixed in a new Keira3 version.</p>
           </div>
```

Comment:
Maybe we should mention here that the Models have been moved to a separate editor